spec/component/importers/activity_insight_importer_spec.rb

```diff
@@ -5029,6 +5031,7 @@
 
           it 'creates a new ActivityInsightOAFile for that publication' do
             expect(DOIVerificationJob).to receive(:perform_later).exactly(3).times
+            expect(AiOAStatusExportJob).to receive(:perform_later).once
```

Comment:
I think we should have a test somewhere in here that tests when the status doesn't update because it is not blank.  We should also test that the post_print_status gets updated in the database.  This would be just for when there's a new file.  I'm not too concerned about tests for the back fill code since it will be removed.